app/auth/reducer.js

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Survey reducer
+ *
+ */
+
+import { fromJS } from 'immutable';
+// import { <insert action> } from './actions';
+import {
+  LOGIN_SUCCESS,
+  LOGIN_ERROR,
+  SET_AUTH_TOKEN,
+  SIGNUP_REQUEST,
+  SIGNUP_SUCCESS,
+  SIGNUP_ERROR,
+  LOGOUT,
+
+} from './constants';
+
+const initialState = fromJS({});
```

Comment:
i prefer when initialState reflects the populated state with `undefined` marking each key